tests/test_profile.py

```diff
@@ -256,7 +277,7 @@ def test_that_user_can_create_and_delete_group(self, base_url, selenium, vouched
 
         home_page = Home(base_url, selenium)
         home_page.login(vouched_user['email'], vouched_user['password'])
-        edit_page = home_page.header.click_edit_profile_menu_item()
+        edit_page = home_page.header.click_settings_menu_item()
         groups = edit_page.click_find_group_link()
```

Comment:
This fails locally since the new Settings class has no longer a `click_find_group_link` method.